crates/python/src/executable.rs

```diff
@@ -93,54 +115,56 @@ impl PyExecutable {
         Self::from(Arc::new(Mutex::new(exe)))
     }
 
-    pub fn execute_on_qvm<'py>(&self, py: Python<'py>) -> PyResult<&'py PyAny> {
-        let arc = self.as_inner().clone();
-        pyo3_asyncio::tokio::local_future_into_py(py, async move {
-            arc.lock()
-                .await
-                .execute_on_qvm()
-                .await
-                .map(ExecutionData::from)
-                .map(|qvm| Python::with_gil(|py| qvm.to_python(py)))
-                .map_err(ExecutionError::from)
-                .map_err(ExecutionError::to_py_err)?
-        })
+    #[pyo3(name = "execute_on_qvm")]
+    pub fn py_execute_on_qvm(&self) -> PyResult<PyExecutionData> {
```

Comment:
All of these methods could have the `py_` prefix and `name = ` removed to get the same effect.